### ISSUE TITLE
servers never online issue

### ISSUE THREAD

**Issue by ianhaney** [phpservermon/phpservermon#756]
I have the latest version of php server monitor installed but the websites servers keep saying last online: Never and showing red on status tab. I have cron job running every 15 minutes but still no joy with it, it looks like it's port 443 on the servers tab.

When click on the server to view the info, it's saying off as the status and says TIMEOUT ERROR: no response from server

can anyone help please?

UPDATE: I have solved it by using http and works but it don't seem to like https for some reason, is there any fix for this?

**Comment by SpcCw** [comment]
I walked into the same yesterday after updating OS to Buster.

It seems like curl PHP library updated and refused to connect to HTTPS. I ended up reinstalling the latest version of PHP Server Monitor (from develop branch here, just downloaded .zip, extracted it over existing installation, run "php composer.phar install" from its directory as readme said and then opened https://myphpmonurl/install.php to complete the update.

It worked fine then.
